tests/manual-test-cases/Group5-Functional-Tests/5-9-Private-Registry.md

```diff
@@ -0,0 +1,28 @@
+Test 5-9 - Private Registry
+=======
+
+#Purpose:
+To verify the VIC appliance can pull an image from a private registry
+
+#References:
+[1 - Docker Registry](https://hub.docker.com/_/registry/)
+
+#Environment:
+This test requires access to a vSphere Server
+
+#Test Steps:
+1. Install a new VCH appliance into the vSphere Server
+2. Start the docker registry locally:  
+```docker run -d -p 5000:5000 --name registry registry```
+3. Pull an image and tag it for the new local registry:  
+```docker tag busybox localhost:5000/busybox:latest```
+4. Push the tagged image to the registry:  
+```docker push localhost:5000/busybox```
+5. Attempt to pull the local registry image using the VCH appliance:  
+```docker pull ${params} 172.17.0.1:5000/busybox```
```

Comment:
No, that last call is switching from the localhost calls to the VCH, that is why ${params} showed up too in that last call.